### PR TITLE
Prepare v1.0.0 release / Format 1A8 release

### PR DESCRIPTION
**Pull request by pksunkara**
This PR is just a preparation for release. The official release will be tagged and released once all the other repositories have been prepared for release.

### CODE REVIEW

**Comment by pksunkara** [comment]
This PR prepares the release for snowcrash version v1.0.0. This is a major version release because we have remove the CLI, serialization and the c-interface from the snowcrash.

File: src/Version.h [hunk 1]
```diff
@@ -14,9 +14,9 @@
 #ifndef SNOWCRASH_VERSION_H
 #define SNOWCRASH_VERSION_H
 
-#define SNOWCRASH_MAJOR_VERSION 0
-#define SNOWCRASH_MINOR_VERSION 16
-#define SNOWCRASH_PATCH_VERSION 1
+#define SNOWCRASH_MAJOR_VERSION 1
```
Comment (zdne) [inline src/Version.h]: Main question is whether we need this versioning here – as it was used mainly for the CLI tool

**Comment by zdne** [comment]
@pksunkara also can you explain why 1.0.0 instead of 0.17.0?

**Comment by klokane** [comment]
And we should probably add function which will return version info for `snowcrash` lib because of split to drafter

**Comment by pksunkara** [comment]
> This is a major version release because we have remove the CLI, serialization and the c-interface from the snowcrash.

@zdne That is the reason I stated for a major version.

**Comment by pksunkara** [comment]
@klokane I don't think we need a function for that since snowcrash is now only a library and not a CLI. `SNOWCRASH_VERSION_STRING` should take care of it.

**Comment by klokane** [comment]
@pksunkara what about C-interface

**Comment by pksunkara** [comment]
What about c-interface? It was removed from snowcrash and moved to drafter, isn't it?

**Comment by klokane** [comment]
point for you :)
But e.g. in `redsnow` you still should be able to know version of `snowcrash` you use

**Comment by pksunkara** [comment]
hmm.. No. Redsnow needs to know the drafter version and probably the API Blueprint AST & Parse Result version. Not the snowcrash version.

**Comment by klokane** [comment]
At least it is good for reporting problems. Instead of investigating "oh this build uses, this drafter and it is builded with snowcrash version xxxxx"

**Comment by pksunkara** [comment]
The point is drafter will contain the snowcrash submodule set at a particular version. So, I think, Drafter should print `SNOWCRASH_VERSION_STRING` when the `--version` option is given.

**Comment by klokane** [comment]
yeah, nice point

**Comment by zdne** [comment]
Well, I am afraid we will have to eventually distinguish between Drafter version and Snow Crash version. As we will start adding features to Drafter e.g. producing JSON and later JSON schema these version can (and should) be different from Snow Crash version. Whereas Snow Crash version is basically saying what from the spec is supported. However ultimately the Drafter version is the important one to the end users not Snow Crash version. So if there should be only one it is Drafter version and not Snow Crash version. Does it make sense?

**Comment by pksunkara** [comment]
> So if there should be only one it is Drafter version and not Snow Crash version. 

I understand. But there's nothing wrong with maintaining a `SNOWCRASH_VERSION_STRING` and printing it along with drafter version when `drafter -v` is called.

**Comment by klokane** [comment]
Agree I mean, there should be reported drafter version primary, and used snowcrash version as additonal info

**Comment by pksunkara** [comment]
Updated. This PR now removes the whole `Version.h` from snowcrash. Snowcrash versions will only be tagged in github, not in the code.

**Comment by zdne** [comment]
:+1: waiting for tests

**Comment by zdne** [comment]
.. the travis seems to be super slow ...

**Comment by pksunkara** [comment]
We should move snowcrash to circle-ci.

**Comment by klokane** [comment]
it depends, sometime it takes long time for appveyor, sometime circle

**Comment by zdne** [comment]
> We should move snowcrash to circle-ci. 

The final decision is up to @Almad and @abtris – to be decided in a week time

**Pull request merged**